src/java_bytecode/library/Makefile

```diff
@@ -0,0 +1,26 @@
+all: org.cprover.jar
```

Comment:
1. Fine with me if it is in separate commits.
2. Reuben is working on  the Java library. This PR is a prerequisite to enable nondet+assume patterns.